### ISSUE TITLE
Feature Request: Deleted message indicator

### ISSUE THREAD

**Issue by Deleora** [johanw666/Signal-Android#22]
Hi,
Is it possible to add some kind of indicators to message bubbles of deleted messages so that we know which message has been deleted by the other person? I think it would really be a great feature and compliment the "Ignore remote delete" feature. What do you think about this?

**Comment by johanw666** [comment]
I agree that it would be usefull, but it needs to fit in my limited time to maintain. I'll look into it.

**Comment by ProblematicDude** [comment]
Up voting. As I replied to a someone's deleted message. Other person was like What the heck. How can you reply to a deleted message.  Made 1000 false stories to justify that it was signal server fault and I'm not using a work around.

**Comment by zezaku** [comment]
Hello Johan and thank you for this fork.
Im using it daily.
I will like to know if u can think about adding these features in the future 

1.Securely shreds sensitive data from RAM
2.Allows you to delete contacts and stop sharing your profile
3.Clears call notifications together with expiring messages
4.Disables debug logs
5.Supports SOCKS proxy and Tor via Orbot

There is another fork of Signal named [Molly](https://github.com/mollyim/mollyim-android)  who does all this but i like your fork more.
Kind regards ,
Alex

**Comment by signalman24-mollybaly** [comment]
I would like to express my support for this idea as well. Not knowing when a message has been deleted causes only problems. Many of the messenger mods that I've tried for other apps have this feature, which I feel is necessary.
The ones i'm talking about displayed an emoji or a short "DEL" text next to the message once it was deleted, letting me know that it has been deleted although I can still see it. Would be great to see this

**Comment by johanw666** [comment]
A deleted message indicator has been implemented in version 5.32.7.1-JW. When a message is remote deleted and the ignore remote delete option is on, an exclamation mark with sender self will be shown.

I thank ClauZ for implementing the function.
